floodlight/io/skillcorner.py

```diff
@@ -0,0 +1,281 @@
+import json
+
+from typing import Dict, Tuple, Union, List
+
+import numpy as np
+import pandas as pd
+
+from floodlight import Teamsheet, Code, XY, Pitch
+
+
+def _read_json(file_path: str) -> Dict:
+    """
+    Read a JSON file from the given file path and return its content.
+
+    Parameters
+    ----------
+    file_path : str
+        The path to the JSON file to be read.
+
+    Returns
+    -------
+    dict
+        The content of the JSON file as a dictionary.
+    """
+    with open(file_path) as f:
+        return json.load(f)
+
+
+def get_meta_data(
```

Comment:
looks more like a private function (``_get_meat_data``) or does it have any value for users? because from the docstring it does not really become evident what ``match_data`` is, or at least how the values for the different keys should look like, so I woulndt be really able to use it